vars/neoDeploy.groovy

```diff
@@ -84,10 +84,15 @@ import static com.sap.piper.Prerequisites.checkScript
      * @see dockerExecute
      */
     'dockerOptions',
+    /**
+      * Extension files. Provided to the neo command via parameter `--extensions` (`-e`).
+      */
+    'mtaExtensionDescriptors',
```

Comment:
I suggest to call the parameter `extensions` like the command option.

Reply:
mmhhhh. Not sure. The extensions are only valid in case of deploy mode `MTA`. From that point of view: isn't it better to have the "mta" in the name. But I updated the documentation accordingly in order to have that also documented.

Reply:
I dislike fighting for parameter names as long the name(s) come with a minimum level of reasonability. This is IMO the case for both names (each having its pros and cons). I switched to your suggestion.

Reply:
ok, thank you.